File: android/app/src/main/java/com/example/android/Logger/LoggerExtensions.kt
package com.example.android.logger

import java.lang.Exception

class LoggerExtensions {
    fun log(logger: ILogger, message: String) {
        logger.Log(LogEntry(LoggingEventType.Information, message))
    }

    fun log(logger: ILogger, ex: Exception) {
        logger.Log(LogEntry(LoggingEventType.Error, ex.message, ex))
    }
}

File: .net/webApi/CreateAndUpdateKmlWebApi/FunctionalTest/Log/LoggerExtensions.cs
namespace FunctionalTest.Log;

public static class LoggerExtensions
{
    public static void Log(this ILogger logger, string message) =>
        logger.Log(new LogEntry(LoggingEventType.Information, message));

    public static void Log(this ILogger logger, Exception? ex)
    {
        if (ex?.Message != null) logger.Log(new LogEntry(LoggingEventType.Error, ex.Message, ex));
    }

    // More methods here.
}